proto/gen.sh

```diff
@@ -8,14 +8,14 @@ protoc -I . \
   --plugin=$TS_PROTO_PATH --ts_proto_out=. --ts_proto_opt=outputClientImpl=false --ts_proto_opt=outputEncodeMethods=false \
   --ts_proto_opt=outputJsonMethods=false --ts_proto_opt=snakeToCamel=false --ts_proto_opt=stringEnums=true \
   --ts_proto_opt=fileSuffix=.pb --ts_proto_opt=useOptionals=fakeValueToOnlyAllowUnionForUndefinedOnOptionalTypes \
-  --go_out=. --go_opt=paths=source_relative \
+  --go_out=. --go_opt=paths=source_relative --experimental_allow_proto3_optional \
```

Comment:
What does this line do? Is it required due to your protoc version being older than mine?